src/web/HTMLCategory.mjs

```diff
@@ -43,7 +43,7 @@ class HTMLCategory {
         <a class="category-title" data-toggle="collapse" data-target="#${catName}">
             ${this.name}
         </a>
-        <div id="${catName}" class="panel-collapse collapse ${(this.selected ? " show" : "")}" data-parent="#categories">
+        <div id="${catName}" class="panel-collapse collapse ${(this.selected ? "show" : "")}" data-parent="#categories">
```

Comment:
`${this.selected && "show"}` might be a cleaner approach than a ternary, considering you don't want anything to be done if the evaluated property is `false`